core/src/load_library.rs

```diff
@@ -29,13 +30,6 @@ const LIBRARY_INSTALL_PATH: &str = "/opt/mongodb/atlas-sql-odbc-driver/";
 static INIT: Once = Once::new();
 static mut MONGOSQLTRANSLATE_LIBRARY: Option<Library> = None;
 
-#[repr(C)]
-pub struct RunCommandResult {
-    result_ptr: *const u8,
-    result_len: usize,
-    result_cap: usize,
-}
-
```

Comment:
I refactored this file to use `LibmongosqltranslateDataIO` instead of `RunCommandResult` because they are ultimately the same thing.

Reply:
More generally, we could  call it `BsonBuffer`. I'm not sure we should be adding "Result" to the name, since the type doesn't reflect that. Thoughts?

Reply:
This is what's currently being used:
```
#[repr(C)]
pub struct LibmongosqltranslateDataIO {
    pub data: *const u8,
    pub length: usize,
    pub capacity: usize,
}
```
I got rid of `RunCommandResult`